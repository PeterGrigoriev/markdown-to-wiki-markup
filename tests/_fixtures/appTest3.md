Sample stress test

1. foo `bar`
1. baz

![An image example](https://i.imgur.com/HbMC7zw.png)