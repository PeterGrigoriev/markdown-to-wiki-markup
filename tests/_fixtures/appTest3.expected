Sample stress test

1. foo `bar`
1. baz

[[Image(https://i.imgur.com/HbMC7zw.png)]]